pages/api/graphql-rest.page.ts

```diff
@@ -15,6 +15,18 @@ const typeDefs = gql`
     taskAnalytics(accountListId: ID!): TaskAnalytics!
   }
 
+  type Mutation {
+    exportContacts(input: ExportContactsInput!): String!
+  }
+
+  input ExportContactsInput {
+    format: String!
+    mailing: Boolean!
+    labelType: String
+    sort: String
```

Comment:
Could either of these be enums?

Reply:
Possibly but none of the examples in the angular codebase ever use these fields for the export so I'm not exactly sure what the values are. The docs don't have anything about them either: https://developers.mpdx.org/api/v2/contacts/exports

Reply:
Actually, I figured this out while working on the last bit of the exporting logic. I've turned these into Enums.